RUFAS/classes.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, config, weather, time):
                 initialize the state
         """
         feed_class_config = im.get_data("feed")
-        self.feed = Feed(feed_class_config)
+        self.feed = Feed(feed_class_config, nutrient_standard=im.get_data("config.nutrient_standard"))
```

Comment:
```suggestion
        nutrient_standard = im.get_data("config.nutrient_standard")
        self.feed = Feed(feed_class_config, nutrient_standard)
```
Not important, but keeps line consistent with above and below lines.